### DISCUSSION TITLE
Auto insert README.md from specific repos of mine as sub pages?

### DISCUSSION THREAD

**Discussion by imthenachoman** [community/community#22008] [Other Feature Feedback, Questions, & Ideas]
<p>I have created a GH Pages site.</p>
<p>I have a slew of repos in GH and each repo has a README.md file.</p>
<p>I am trying to find a way to have those README.md files load on my GH Pages site? That way as I update my repo README.md file, the updates show up in my GH pages.</p>
<p>Is this possible?</p>

**Comment by jfullstackdev** [comment]
<p>all of those sub READMEs are automatically turned to a GitHub Page too but unfortunately, what I’m seeing in order for that to appear at the landing page is to create your custom <code>index.html</code> so it’s your choice how you will be doing it like embedding it or simply providing the links at the default README so they will know there is the sub README at your page</p>
<p>if you feel it can’t be managed that way, that’s the time to use the <code>docs</code> folder or <code>gh-pages</code> branch</p>

**Comment by imthenachoman** [comment]
<p>The READMEs from OTHER projects/repos are turned into GH pages?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>They would be if you were to enable GH Pages for those repositories, too.</p>
<p>If you want them all automatically included in one Pages site you’ll have to build the automation yourself, e.g. an Actions workflow running on a schedule that checks if there have been updates to any of the READMEs, collects them in whatever way you want, and pushes to the branch you use for GH Pages.</p>
